applications/nrf_desktop/doc/dfu.rst

```diff
@@ -132,6 +145,8 @@ The device generation allows to distinguish configurations that use the same boa
 reboot
    Perform the fetch operation on this option to trigger an instant reboot of the device.
 
+   With the :ref:`CONFIG_DESKTOP_DFU_LOCK <config_desktop_app_options>` option enabled, the device only executes this command when it holds the DFU owner status.
```

Comment:
```suggestion
   The device executes this command, only if the :ref:`CONFIG_DESKTOP_DFU_LOCK <config_desktop_app_options>` option is enabled, and it holds the DFU owner status.
```
Hope the meaning has not changed.

Reply:
It has. With the DFU lock disabled, the command will execute no matter the DFU owner status.

```suggestion
   If the :ref:`CONFIG_DESKTOP_DFU_LOCK <config_desktop_app_options>` option is enabled, the device executes this command, only if it holds the DFU owner status.
```